src/table.js

```diff
@@ -604,44 +645,36 @@ Table.prototype.delete = function(gaxOptions, callback) {
  *   }
  * };
  *
- * table.deleteRows({
- *   prefix: 'alincoln'
- * }, callback);
- *
- * //-
- * // If you choose to omit the prefix, all rows in the table will be deleted.
- * //-
- * table.deleteRows(callback);
+ * table.deleteRows('alincoln', callback);
  *
  * //-
  * // If the callback is omitted, we'll return a Promise.
  * //-
- * table.deleteRows().then(function(data) {
+ * table.deleteRows('alincoln').then(function(data) {
  *   const apiResponse = data[0];
  * });
  */
-Table.prototype.deleteRows = function(options, callback) {
-  if (is.function(options)) {
-    callback = options;
-    options = {};
+Table.prototype.deleteRows = function(prefix, gaxOptions, callback) {
+  if (is.function(gaxOptions)) {
+    callback = gaxOptions;
+    gaxOptions = {};
+  }
+
+  if (!prefix || typeof prefix === 'function') {
+    throw new Error('A prefix is required for deleteRows.');
```

Comment:
We can note in the JSDocs block:

```js
@throws {error} If a prefix is not provided.
```

(reference: https://github.com/kolodny/nodejs-bigtable/blob/95e626dd9b7c91cca4f9f11ac3f8c0953c64e22b/src/table.js#L178)

Reply:
Done